python/testing/TestPythonCompareWithFile.py

```diff
@@ -6,6 +6,8 @@
 output = sys.argv[2] + "TestPythonCompareWithFile.png"
 outputDiff = sys.argv[2] + "TestPythonCompareWithFile.diff.png"
 
+f3d.engine.loadPlugin("core")
```

Comment:
should nt that be added to all our SDK tests, C++ and python ?

Reply:
For C++, it is automatically done:
```cmake
set(CMAKE_TESTDRIVER_BEFORE_TESTMAIN "f3d::engine::loadPlugin(\"core\");")
```

For other python tests, we are not reading any files.

Reply:
ok